### PR TITLE
[Storage] Add breaking change type

### PR DESCRIPTION
**Pull request by blueww**
## Description

Add some breaking change for output type, and remove the before added breaking change warning message
target to 10/27 release.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,17 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+*Removed obsolete property RestorePolicy.LastEnabledTime
```
Comment (isra-fel) [inline src/Storage/Storage.Management/ChangeLog.md:21]: nit: format
```suggestion
* Removed obsolete property RestorePolicy.LastEnabledTime
```

**Review by isra-fel — CHANGES_REQUESTED** [review]
Generally LGTM 
You'll need to suppress the detected breaking change errors, see https://github.com/Azure/azure-powershell/blob/master/documentation/Debugging-StaticAnalysis-Errors.md#breaking-changes

**Review by isra-fel — APPROVED** [review]

**Review by wyunchi-ms — DISMISSED** [review]

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**